src/services/appointment.service.ts

```diff
@@ -83,12 +84,17 @@ export default class AppointmentService implements OnModuleInit {
       customer,
       treatments,
     });
-
-    let savdAppt;
+    const eventId = await this.googleCalendarService.createEvent(appointment);
```

Comment:
💭 After discussion, a few points were highlighted here:
- If the user is not authenticated the events in the calendar are going to be lost, so the user needs to be authenticated all the time
- In order to backfill all of the missing events, a corn job needs to be scheduled in order to add any missing event

This is just a note, no changes required here. But make sure to account for these scenarios in future tickets since this is a core logic for the application